tesseract_monitoring/src/robot_comm_monitor_node.cpp

```diff
@@ -0,0 +1,97 @@
+/*
+ * robot_communication_monitor.cpp
+ *
+ * Checks for fanuc communication errors
```

Comment:
There does not seem to be anything Fanuc specific here, no.

The node subscribes to `joint_states` and then keeps resetting a timeout upon reception of msgs. No msgs -> publish a `Bool` on a specific topic.

This would work for any ROS driver that publishes `JointState`s.